### ISSUE TITLE
Rendering the openAPI specs

### ISSUE THREAD

**Issue by m-mohr** [radiantearth/stac-spec#249]
It would be nice to have rendered versions of the API spec that you could directly link to.

**Comment by hgs-truthe01** [comment]
@cholmes is using swaggerhub to host the api docs.

https://app.swaggerhub.com/apis/cholmesgeo/STAC-standalone/0.5.1

There is a link to it on the api readme, but it is hard to find if you don't know it is there. Maybe we should make that link more visible?

https://github.com/radiantearth/stac-spec/tree/master/api-spec
The link is in the Specification section.

**Comment by m-mohr** [comment]
Thanks, that's a great start, but for the current state in dev there is no generated version, right? It would make easier to share ideas and discuss things currently in work with partners, collegues and so. For openEO we use the CI to generate the docs and host them on Github Pages. Could that be an option here, too?

**Comment by cholmes** [comment]
I'm definitely super happy for a CI generated doc site. I used swaggerhub as it was the quickest way to get what I wanted. I guess one thing is we'd need a location on the web - may make more sense after we have #25 implemented (I haven't been pushing on it since the core spec was more important, but I think it's getting closer to being a priority - once we have landsat and sentinel datasets implemented as STAC and on STAC Browser I think is when it'll bubble to the top of my list).

Definitely happy to have the link more visible. I had thought it was, but on my latest structure update I agree it's a bit buried - we can pull it out and put it at the top of the document. Until we have generated docs we should just use swaggerhub - ipdating the swaggerhub should be part of the release process and I do try to do it. May have to figure out how to let other people update it if others are going to help out on releases.
